DatabaseConnectionProvider.csproj

```diff
@@ -1,19 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
-
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>

```

Comment:
pytanie na boku bawimy się w wersjonowanie?

Reply:
You mean versioning of our whole backend app? Well we can, but versions will be modified while develop will be merged into a master branch. It's not up to me to specify or change project version now. We should discuss that on next meeting, and if we agree to that. Version can be specify herel like I said, before master branch merge.